build.gradle

```diff
@@ -11,7 +11,7 @@ sourceCompatibility = 17
 targetCompatibility = 17
 
 def remoteRobotVersion = "0.11.18"
-def lsp4ijVersion = '0.0.2-20240615-013020'
+def lsp4ijVersion = '0.0.2-20240615-013020@nightly'
```

Comment:
Adding a comment explaining how to configure a release build and a stable build in this variable would be good.

Reply:
Added the comment